src/lib/chips/chip-list.spec.ts

```diff
@@ -242,3 +243,19 @@ class StaticChipList {
   chipDeselect(index: Number) {
   }
 }
+
+@Component({
+  template: `
+    <md-input-container>
```

Comment:
cc @mmalerba - thoughts?

I'm not sure putting the chip-list inside of the input-container is the approach we ultimately want to take. Right now I don't think the input-container is in itself designed to have any content other than the input/textarea and friends. 

I know we want to make the chips be part of the input's underline, but perhaps there's another way we could approach that.